src/Unused/Parser/Internal.hs

```diff
@@ -0,0 +1,45 @@
+module Unused.Parser.Internal
+    ( parseTermMatches
+    , parseTermMatch
+    , parse
+    , ParseError
+    ) where
+
+import Control.Monad (void)
+import Text.Parsec
+import Text.Parsec.String (Parser)
+import Unused.Types (TermMatch(..))
+
+parseTermMatches :: Parser [TermMatch]
+parseTermMatches = many1 parseTermMatch <* eof
+
+parseTermMatch :: Parser TermMatch
+parseTermMatch = do
+    colonSep
+    path <- pathParser
+    colonSep
+    occurrences <- occurrenceParser
+    colonSep
+    term <- termParser
+    void eol
+
+    return $ TermMatch term path $ toInt occurrences
+  where
+    toInt i = read i :: Int
```

Comment:
May want to use `readMaybe` here, unless there's some reason `read` should always work. http://hackage.haskell.org/package/base-4.8.2.0/docs/Text-Read.html#v:readMaybe